File: PythonFiles/uberB.py
from uberCar import uberCar

class uberB:
    typeCarAccepted = str
    seatsMaterial = str

    def __init__(self, license,driver,passengers, typeCarAccepted, seatsMaterial):
        super.__init__(license,driver,passengers)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial

File: JavaFiles/UberB.java
package JavaFiles;

class uberB extends uberCar
{
    String license;
    Account driver;
    
    

 public uberB(String license, Account driver)
 {
     super(license, driver);
     this.license = license;
     this.driver = driver;

 }  
 @Override
 void printDataCar() {
     super.printDataCar();
 }
}